response.go

```diff
@@ -195,9 +257,13 @@ func (r RedirectResponse) GetContentType() string {
 	return "text/html; charset=utf-8"
 }
 
+func (r RedirectResponse) GetContent() any {
```

Comment:
if you have to return meaningless values for a function, just to implement an interface, than the interfacing is being too narrow. instead, functions that want to call GetContent() should just check the underlying type of the interface.